src/tests/global_variables/env_variables_defaults.py

```diff
@@ -96,6 +96,7 @@ class _EnvVariables(DataPool, ABC):
     hold_installation: EnvVar = EnvVar(
         ["HOLD_INSTALLATION"], loader=lambda x: bool(strtobool(x)), default=env_defaults.DEFAULT_HOLD_INSTALLATION
     )
+    reclaim_hosts: EnvVar = EnvVar(["RECLAIM_HOSTS"], loader=lambda x: bool(strtobool(x)), default=False)
```

Comment:
Need to add `RECLAIM_HOSTS` also to `skipper.env`

Reply:
Added, thank you!